src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -441,18 +435,49 @@ protected function getQueries($migration, $method)
     }
 
     /**
-     * Resolve a migration instance from a file.
+     * Resolve a migration instance from migration name.
      *
-     * @param  string  $file
+     * @param  string  $migrationName
      * @return object
      */
-    public function resolve($file)
+    public function resolve($migrationName)
```

Comment:
As Laravel 8.x supports PHP 8, changing a public method parameter name is a breaking change, due to named parameters feature introduced in PHP 9

Reply:
@rodrigopedra changing parameter names is an exception to our BC policy: https://laravel.com/docs/8.x/releases#named-arguments

Reply:
That being said, yes please revert this.

Reply:
Thanks @driesvints , I was not aware of that

Reply:
@driesvints I'm confused, should I keep `$migrationName` or revert to `$file`. As I mentioned before, what is being passed to this method is not a file, but a migration name.

Reply:
@thiagorb let's please focus on the feature itself without too much distraction.

Reply:
The paramter's name was reverted.